spec/classes/role/www_lib_vm_spec.rb

```diff
@@ -263,6 +257,26 @@
           .with_error_log_file('error.log')
           .with_custom_fragment(%r{CookieName skynet})
       end
+
+      # Files
+      it { is_expected.to contain_file('sqlnet.ora') }
+
+      it 'adds custom params to file authz_umichlib.conf' do
+        is_expected.to contain_file('authz_umichlib.conf')
+          .with_content(%r{DBDParams\s*user=somebody})
```

Comment:
Haha, I was about to comment, saying I was confused about why this was being deleted (took me a minute to realize you just moved and improved it). This is definitely better than

```ruby
it do
  # set via hiera
```